### DISCUSSION TITLE
What package to specify in Cake 2.0 since Cake.CoreCLR has been deprecated

### DISCUSSION THREAD

**Discussion by nick5454** [cake-build/cake#3740] [Extension Q&A] [answered]
I switch my ps to 

`dotnet new tool-manifest

dotnet tool install Cake.Tool

dotnet cake`
But how do I change it to run a custom file name instead of build.cake? when I run dotnet cake -h I don't see a command to run a custom filename

**Comment by nick5454** [comment, marked as answer]
dotnet cake recipe.cake

**Comment by pascalberger** [comment]
See [Cake .NET Tool usage](https://cakebuild.net/docs/running-builds/runners/dotnet-tool#usage):

```
dotnet cake [script] [switches]
```
